models/public/midasnet/midasnet.md

```diff
@@ -0,0 +1,100 @@
+# midasnet
+
+## Use Case and High-Level Description
+
+MidasNet is a model for monocular depth estimation trained by mixing several datasets;
+as described in the following paper:
+"Towards Robust Monocular Depth Estimation: Mixing Datasets for Zero-Shot Cross-Dataset Transfer"
+<https://arxiv.org/pdf/1907.01341>
+
+The model input is a blob that consists of a single image of "1x3x384x384" in `BGR` order.
+
+The model output is an inverse depth map that is defined up to an unknown scale factor.
+
+## Example
+
+See [here](https://github.com/intel-isl/MiDaS)
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Monodepth     |
+| GFLOPs            | ?             |
+| MParams           | ?             |
+| Source framework  | PyTorch\*     |
+
+## Input
+
+### Original Model
+
+Image, name - `data`, shape - `1,3,384,384`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
```

Comment:
Please, add here mean and scale value if any. But only for *original model*.

The main idea of this section is to provide information about which input is expected, and what kind of preprocessing should be done.

Reply:
man and scale values added